tf-modules/archive/variables.tf

```diff
@@ -10,6 +10,12 @@ variable "background_queue_url" {
   type = string
 }
 
+variable "cloudwatch_log_retention_periods"{
+  type = map(string)
+  description = "number of days logs will be retained for the respective cloudwatch log group, in the form of module_<cloudwatch_log_group_name>_log_retention"
```

Comment:
Nitpick, but `module_<cloudwatch_log_group_name>_log_retention` should probably be `<module>_<cloudwatch_log_group_name>_log_retention` so users don't think it's the literal string "module".